docs/requirements.txt

```diff
@@ -1 +1,2 @@
-sphinx-rtd-theme==1.3.0
+sphinx
+sphinx-rtd-theme
```

Comment:
I'm not much of a fan on unpinned Python packages. How static is the Sphinx format? Is there likely to be any breaking change that will stop the doc generation from working?

To avoid needing to bump patch versions we can specify the version using a [compatible release clause](https://peps.python.org/pep-0440/#compatible-release) (e.g. `sphinx-rtd-theme~=1.3.0`) ensuring that any future patch versions are included if they get created.